README.md

```diff
@@ -61,12 +79,15 @@ for a second time (based on timestamps and hashes)
 - starts an AWS instance, and waits until it's ready (or just runs the
 execution locally depending on the configuration)
 - streams the logs the same as if you were running your program directly
+- shows metrics you collected during the run, such as accuracy and loss (you
+can query those later)
+- downloads output files you might have created.
 
-You can be patient and wait until it works, but you can also hit `Ctrl-C` and
-stop the program early:
+You can be patient and wait until it finishes, but you can also hit `Ctrl-C`
+and stop the program early:
 
 ```
-k: 0.5209
+Epoch: 9 Traning loss: 0.330538
```

Comment:
Spelling: Training and not Traning

Also pls try to put some space between _Epoch: 9_ and _Training loss:..._

Reply:
Done